### PR TITLE
Add reqpart to docs

### PR DESCRIPTION
**Pull request by pbokoc**
Add a description of the reqpart command to kickstart-docs.rst.

The command was implemented in rhbz#1164660.

### CODE REVIEW

File: docs/kickstart-docs.rst [hunk 1]
```diff
@@ -1783,6 +1783,19 @@ cause a conflicting repo error.
     after reboot. Added in anaconda-22.3-1
 
 
+reqpart
+-------
+
+Automatically create partitions required by your hardware platform. These include a ``/boot/efi`` for x86_64 systems with UEFI firmware, ``biosboot`` for x86_64 systems with BIOS firmware and GPT, and ``PRePBoot`` for IBM Power Systems.
```
Comment (bcl) [inline docs/kickstart-docs.rst]: I don't think these are limited to x86_64. It should also work on aarch64 UEFI systems for example.

**Comment by pbokoc** [comment]
Added Aarch64 UEFI to the description.

**Comment by vpodzime** [comment]
Looks good to me. Could you please squash the two commits into a single one?

**Comment by pbokoc** [comment]
Squashed.

**Pull request merged**